redbeat/schedulers.py

```diff
@@ -191,11 +191,13 @@ def __init__(self, app, **kwargs):
     def setup_schedule(self):
         # cleanup old static entries
         client = redis(self.app)
-        previous = client.smembers(self.app.conf.REDBEAT_STATICS_KEY)
-        current = set(self.app.conf.CELERYBEAT_SCHEDULE.keys())
-        removed = previous - current
+        previous = set(key.decode('utf-8')
```

Comment:
This means I need to do another PR introducing `decode_responses`, since some of the Python 3 compatibility fixes need to be adapted, and the `fakeredis` version needs to be pinned to `>=0.8.1` in the requirements.